flatpak/me.broken_by.PdfMetadataEditor.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with
+  jdAppStreamEdit 7.1-->
+  <id>me.broken_by.PdfMetadataEditor</id>
+  <name>Pdf Metadata Editor</name>
+  <summary>Edit PDF metadata</summary>
+  <developer_name>Svetlozar Argirov</developer_name>
+  <metadata_license>GFDL-1.3</metadata_license>
```

Comment:
CC0-1.0 is prefered for the metadata.

This file should also be managed upstream for future releases.

Reply:
just to had, this file is NOT flatpak specific either.

Reply:
Changed